Contributing.md

```diff
@@ -18,6 +18,18 @@ As an organization contributor, you can merge other people's pull requests, or o
 
 This project is not continuously deployed, there is space for debate after review too. Offering everyone the chance to revert, or make an amending pull request. If it feels right, merge.
 
+#### Bug fixes and Release cycles
```

Comment:
Maybe we could make the section header something like **Development Process**?

Reply:
Then again, we have [Development.md](https://github.com/Moya/Moya/blob/master/docs/Development.md) already in place, maybe move it there? 🤔